src/components/subnavigation/developer-topics-menu.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useContext } from 'react';
+import { Context } from './navbar';
+import { DeveloperTopicsMenuWrapper } from '../../styled/developer-topics-menu';
+import { useRouter } from 'next/router';
+
+const DeveloperTopicsMenu: React.FunctionComponent = () => {
+    const { open } = useContext(Context);
+    const router = useRouter();
+
+    const handleDeveloperTopicsClick = () => {
+        router.push('/developertopics').then(r => {
+            console.log(r);
+        });
+    };
+
+    return (
+        <>
+            {open && (
+                <DeveloperTopicsMenuWrapper>
+                    <button onClick={handleDeveloperTopicsClick}>
+                        All Developer Topics
+                    </button>
```

Comment:
Would we be able to just wrap the button in an anchor tag with an `href="/developertopics"` rather than use the next router manually? I'm not positive but I think it may be better for SEO to have anchor links when we can?

Reply:
Agree with you. Only used Router here to bring up discussion on it. Not sure what is the advantage of Next/Router over Links and a tags.

Reply:
Yeah I didn't know it was an option, I guess it could be cool for more complex routing in the future so good to know it's there.